.travis.yml

```diff
@@ -13,6 +13,9 @@ matrix:
   fast_finish: true
 env:
   global:
-  - TEST_REPO=
-  - CORE_REPO=
-  - GEM_REPOS=
+  - TEST_REPO=manageiq#19378
```

Comment:
This should be `CORE_REPO` since you want to use it across all of your matrix tests

Reply:
Oops, thanks.